src/drivers/intel/ssp/ssp.c

```diff
@@ -943,6 +958,12 @@ static void ssp_stop(struct dai *dai, int direction)
 	    ssp->state[SOF_IPC_STREAM_PLAYBACK] == COMP_STATE_PREPARE) {
 		if (!(ssp->params.clks_control &
 			 SOF_DAI_INTEL_SSP_CLKCTRL_BCLK_ES)) {
+
+			/* clear TRSE/RSRE before SSE */
+			ssp_update_bits(dai, SSCR1,
+					SSCR1_TSRE | SSCR1_RSRE,
+					0);
+
```

Comment:
This already cleared above?

Reply:
yes, but this isn't good. we should only deal with RSRE and TSRE along with SSE. The  SSRSA_RXEN and SSRSA_RXEN are the only one that needs to be modified above.

I'll correct this in an update, thanks for the comment

Reply:
fixed